.github/workflows/open-source-unit-tests.yml

```diff
@@ -20,12 +20,10 @@ on:
       image_tag:
         description: 'Marqo image tag. Examples: "1.1.0", "test" "latest"'
         required: true
-  push:
```

Comment:
you can add `workflow_call:` here so this workflow can be directly called by the publish PyPI workflow

Reply:
Good idea. Added, thanks!